Realm/Realm.Unity/Editor/Extensions/MethodReferenceExtensions.cs

```diff
@@ -49,4 +49,25 @@ public static MethodReference MakeHostInstanceGeneric(this MethodReference @this
 
         return reference;
     }
+
+    public static bool HasSameSignatureAs(this MethodReference @this, MethodReference other)
```

Comment:
I wrote in the description of the PR, the third block of text, that I wasn't sure about this one. I wrote that it's not currently used but it made sense in my mind to add since other extension methods of other types have similar features exposed.
If it's better to keep the PR to the minimum necessary for the feature then I'll remove it.

Reply:
Ah, I must have missed that. We generally don't want to have any unused code in the codebase - if at some point a method like this one is needed, whoever needs it will add it.

Reply:
Ok. I'll remove it, then.